src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -52,6 +54,14 @@ public class GetAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         [ValidateNotNullOrEmpty]
         public BackupManagementType BackupManagementType { get; set; }
 
+        /// <summary>
+        /// Policy of the item to be returned.
```

Comment:
Fix the summary string to reflect the intended use of this parameter for this cmdlet.